composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewState.kt

```diff
@@ -0,0 +1,10 @@
+package com.majotyler.hiittimer.presentation
+
+/**
+ * An encapsulation of the UI state required to render a timer which may be incremented.
+ *
+ * @param seconds The number of seconds which the timer has been incrementing for.
+ */
+data class TimerViewState(
+    val seconds: Int,
+)
```

Comment:
If we want to show the time not as seconds, but as a String, like "00:00" we will probably want to change this parameter to be a String. We don't want the `TimerScreen` View to have to deal with any formatting. That is too complicated for a View to deal with.